_kbarticles/2021-05-20-getting-rds-version-information.md

```diff
@@ -0,0 +1,12 @@
+---
+layout: post
+title: "Getting RDS version information"
+date: May 20, 2021
+excerpt: Find out what version of AWS RDS version your app is using
```

Comment:
@ccostino Have I missed anything here?

Reply:
No, this is short and sweet, thank you!  The only thing I'd do is just list the commands for each DB engine we support (so that'd be adding the command for MySQL and Oracle it looks like) in a list to make it easier for folks, but otherwise this is fine as is.